specification/communication/data-plane/JobRouter/preview/2022-07-18-preview/communicationservicejobrouter.json

```diff
@@ -1065,6 +1087,11 @@
         ],
         "summary": "Un-assign a job.",
         "operationId": "JobRouter_UnassignJobAction",
+        "consumes": [
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
```

Comment:
Better to only support "application/json".

Reply:
Have removed everything besides application/json